server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -145,6 +145,20 @@ public CompletableFuture<Token> deleteToken(ServiceRequestContext ctx,
                             .thenApply(unused -> token.withoutSecret()));
     }
 
+    /**
+     * DELETE /tokens/{appId}/removed
+     *
+     * <p>Purges a token of the specified ID that was deleted before.
+     */
+    @Delete("/tokens/{appId}/deleted")
+    public CompletableFuture<Token> purgeToken(ServiceRequestContext ctx,
+                                               @Param String appId,
+                                               Author author, User loginUser) {
+        return getTokenOrRespondForbidden(ctx, appId, loginUser).thenCompose(
+                token -> mds.purgeToken(author, appId)
+                            .thenApply(unused -> token.withoutSecret()));
+    }
```

Comment:
This callback could be called by an event loop if the token is cached.
`mds.purgeToken(author, appId)` is is a blocking operation that must not be called in an event loop. Should we specify `ctx.blockingTaskExecutor()` to the callback using `thenComposeAsync()`?